.github/workflows/artifactLinks.yml

```diff
@@ -0,0 +1,20 @@
+name: Artifact links comments creator
+on:
+  workflow_run:
+    workflows: ["CI"]
```

Comment:
Should be the "PR" workflow if we only want to use with pull requests

Reply:
I think we should use that also on linked issues, is also really useful.

Reply:
How I can add it to PR/CI workflow?

Reply:
```suggestion
    workflows: ["PR"]
```